README.md

```diff
@@ -21,7 +21,15 @@ Currently the following variables are supported:
 
 ### General
 
-* `localectl_var_name` - var\_name description
+* `localectl_system_locale` - Defaults to LANG=en_US.UTF-8. Use this to set the desired language System locale designation for your system.
+
+* `localectl_vc_keymap` - Defaults to US.  Use this to set the the desired VC Keymap designation for your system.
+
+* `localectl_x11_layout` Defaults to US.  Use this to set the desired X11 Layout designation for your system.
```

Comment:
Varname should probably `localectl_x11_keymap` to correspond to the underlying `localctl set-x11-keymap` subcommand.